api/management/api.go

```diff
@@ -12,6 +12,7 @@ type UserRepresentation struct {
 	Label         *string `json:"label,omitempty"`
 	Gender        *string `json:"gender,omitempty"`
 	BirthDate     *string `json:"birthDate,omitempty"`
+	Groups         *[]string `json:"group,omitempty"`
```

Comment:
This line seems not to be aligned like others... Formatting issue ?

Reply:
Done